website/docs/r/datadog_monitors.html.markdown

```diff
@@ -0,0 +1,146 @@
+---
+subcategory: "Datadog"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_datadog_monitor"
+description: |-
+  Manages a datadog Monitor.
+---
+
+# azurerm_datadog_monitor
+
+Manages a datadog Monitor.
+
+## Example Usage
+
+###  Monitor creation with linking to Datadog organization
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-datadog"
+  location = "West US 2"
+}
+resource "azurerm_datadog_monitor" "example" {
+  name                = "example-monitor"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+  datadog_organization {
+    api_key         = "XXXX"
+    application_key = "XXXX"
+  }
+  user {
+    name  = "Example"
+    email = "abc@xyz.com"
+  }
+  sku_name = "Linked"
+  identity {
+    type = "SystemAssigned"
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this datadog Monitor. Changing this forces a new datadog Monitor to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the datadog Monitor should exist. Changing this forces a new datadog Monitor to be created.
+
+* `location` - (Required) The Azure Region where the datadog Monitor should exist. Changing this forces a new datadog Monitor to be created.
+
+* `sku_name` - (Required) The name which should be used for this sku.
+
+* `identity` - (Required) A `identity` block as defined below.
+
+* `user` - (Required) A `user` block as defined below.
+
+* `datadog_organization` - (Required) A `datadog_organization` block as defined below.
+
+---
```

Comment:
```suggestion
```

We only want to use this when splitting out blocks. Having Required/Optional attributes together is preferred

Reply:
Incorporating in next commit.